Mathlib/Algebra/Lie/Nilpotent.lean

```diff
@@ -588,7 +590,13 @@ theorem LieModule.isNilpotent_of_top_iff :
   Equiv.lieModule_isNilpotent_iff LieSubalgebra.topEquiv (1 : M ≃ₗ[R] M) fun _ _ => rfl
 
 @[simp] lemma LieModule.isNilpotent_of_top_iff' :
+    letI : LieRingModule L (⊤ : LieSubmodule R L M) :=
```

Comment:
```suggestion
    #adaptation_note /-- After lean4#5020, many instances for Lie algebras and manifolds are no
    longer found. -/
    letI : LieRingModule L (⊤ : LieSubmodule R L M) :=
```
I don't know if this works. Maybe it has to be an ordinary comment. If so, I suggest leaving the string `#adaptation_note` inside the comment.